app/pb_kits/playbook/pb_online_status/_online_status.jsx

```diff
@@ -4,24 +4,34 @@ import React from 'react'
 import classnames from 'classnames'
 import { globalProps } from '../utilities/globalProps.js'
 
-import { buildDataProps } from '../utilities/props'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 
 type OnlineStatusProps = {
+  aria?: object,
   className?: string,
   data?: object,
   id?: string,
   status?: "online" | "offline" | "away",
 }
 
 const OnlineStatus = (props: OnlineStatusProps) => {
-  const { id, data = {}, className, status = 'offline' } = props
+  const {
+    aria = {},
+    className,
+    data = {},
+    id,
+    status = 'offline',
+  } = props
+
+  const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
-  const css = classnames([`pb_online_status_kit_${status}`, className], globalProps(props))
+  const classes = classnames(buildCss('pb_online_status_kit', status), className, globalProps(props))
```

Comment:
```suggestion
  const classes = classnames(buildCss('pb_online_status_kit', status),globalProps(props), className)
```

Reply:
@jasoncypret Ok, I'll make the change and push it up right away. But if this is the case--that the `className` needs to be at the end--we will need to change most of the kits because `globalProps(props)` was added to the end of all of the class-building code and therefore `className` won't be at the end.  Does that make sense?